build.sbt

```diff
@@ -74,7 +74,10 @@ def app(name: String) =
     .settings(buildDebSettings)
 
 val api = app("membership-attribute-service")
-  .settings(libraryDependencies ++= apiDependencies)
+  .settings(
+    libraryDependencies ++= apiDependencies,
+    dependencyOverrides ++= depOverrides,
+  )
```

Comment:
aws client brings in 2.12 of jackson-databind, but plaay 2.8 brings in 2.11.  Both of those aare security patched, but 2.11 is not happy to use the 2.12 jars at runtime.
```
com.typesafe.play:play_2.12:2.8.8                  (depends on 2.11.4)
software.amazon.awssdk:sdk-core:2.16.69            (depends on 2.12.3)
```
however now it just bombs out at runtime if there is a  binary conflict
`com.fasterxml.jackson.databind.JsonMappingException: Scala module 2.11.4 requires Jackson Databind version >= 2.11.0 and < 2.12.0`